packages/tradershub/src/features/cfd/modals/JurisdictionModal/JurisdictionModal.tsx

```diff
@@ -52,9 +50,6 @@ const JurisdictionModal = () => {
         setCfdState({ selectedJurisdiction });
     }, [selectedJurisdiction, setCfdState]);
 
-    // Do this later: Add Loading Placeholder
```

Comment:
@thisyahlen-deriv Why did you remove this loader?

Reply:
Because it was showing Loading..., and it looked weird

Reply:
It doesnt need a loader since, alll MT5 calls are loaded in before we show the get button